<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dilution Solution</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>164</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="desConcLabel">
      <property name="text">
       <string>Desired Concentration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="desConcValue"/>
    </item>
    <item>
     <widget class="QListWidget" name="desConcUnits">
      <item>
       <property name="text">
        <string>Mol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/mL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/uL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/mL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/uL</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="desVolLabel">
      <property name="text">
       <string>Desired Volume</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="desVolValue"/>
    </item>
    <item>
     <widget class="QListWidget" name="desVolUnits">
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pL</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>164</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="stockConcLabel">
      <property name="text">
       <string>Stock Concentration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stockConcValue"/>
    </item>
    <item>
     <widget class="QListWidget" name="stockConcUnits">
      <item>
       <property name="text">
        <string>Mol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/mL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mMol/uL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/L</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/mL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uMol/uL</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Volume Stock Solution:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Liters:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>uL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Volume Solution:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Liters:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>uL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
